conf/defaults.ini

```diff
@@ -725,12 +725,18 @@ global_alert_rule = -1
 
 #################################### Unified Alerting ####################
 [unified_alerting]
+# Enable unified alerting
+enabled = false
```

Comment:
we can still merge this PR for v8.2 as long as this is false (and we have some backward compatibility with the old feature toggle so users who are early adopters of unified alerting loose all their alerts)

Reply:
@torkelo by backward compatibility you mean that during loading the configuration we discover that the `ngalert` feature toggle was previously enabled and in that case enable the unified alerting (in case somebody has missed setting the new flag)?

Reply:
yea, well what would happen if you missed this new setting and it would revert back to old alerting? if I understood @kylebrandt correctly if you revert back to old alerting and then back to new alerting you will loose all your unified alert rules as it will trigger the migration from old to new again. That feels like a bad experience for users upgrading that miss that this feature toggle has been renamed/changed

Reply:
yes, that's correct: if one reverts to the old alerting, they will loose all their Grafana 8 Alerts and with the present modifications if one misses setting the new setting, they will indeed end up loosing them. I will make the necessary changes.

Reply:
I have pushed the changes, however note that, in `8.2`, it will be launched also [this](https://github.com/grafana/grafana/pull/37414) which forces the migration to re-run so that the Alertmanager configurations are created correctly for each organisations.